src/services/consent/hooks/consents.js

```diff
@@ -134,7 +134,7 @@ const accessCheck = (consent, app) => {
 				requiresParentConsent = false;
 				return Promise.resolve;
 			}
-			const { age } = user;
+			const { age } = user.age;
```

Comment:
does user.age.age really exist? @janrenz

Reply:
old code was `let age = user.age;`. So yes, should exist

Reply:
The age is calculate and added over a hook.